### PR TITLE
DOC bug fixes - TOCs missing from docs

### PR DESCRIPTION
**Pull request by ljamen**
Fixes missing TOC issues in #4673

### CODE REVIEW

File: docs/se/dbclient.adoc [hunk 1]
```diff
@@ -24,6 +24,16 @@
 
 include::{rootdir}/includes/se.adoc[]
 
+==Contents
```
Comment (tjquinno) [inline docs/se/dbclient.adoc:27]: I don't know if either AsciiDoctor or our tool chain (or neither) requires a space between the `==` and the header text. Maybe double-check that this renders properly with a local build?

**Review by tjquinno — COMMENTED** [review]
One question.

File: docs/se/dbclient.adoc (hunk 1 above)
Comment (ljamen) [reply docs/se/dbclient.adoc]: A typo on my part.

**Review by tjquinno — APPROVED** [review]

**Comment by barchetta** [comment]
This PR passed pipeline previously (and one previous failure was in a test not related to docs). Therefore going to force merge.

**Pull request merged**